e2e/wdio/core/tests/wizard.e2e-spec.ts

```diff
@@ -0,0 +1,232 @@
+import { WizardPo } from '../pages/wizard.po';
+import {
+    click,
+    getElementClass,
+    getElementArrayLength,
+    refreshPage,
+    isElementDisplayed,
+    getAttributeByName,
+    doesItExist,
+    setValue,
+    getText,
+    scrollIntoView,
+    getValue,
+    isElementClickable,
+    sendKeys,
+    pause,
+    waitForElDisplayed
+} from '../../driver/wdio';
+import {
+    fullName,
+    firstAdress,
+    secAdress,
+    update,
+    firstAdressLength
+} from '../fixtures/testData/wizard.tags'
+
+describe('Wizard component test', function () {
+    const wizardPage = new WizardPo();
+    const {
+        defaultExample,
+        customizableExample,
+        mobileExample,
+        branchingExample,
+        dialogExample,
+        ngforExample,
+        button,
+        exitButton,
+        wizard,
+        step,
+        nextStep,
+        dialogWizard,
+        fullNameInput,
+        firstAdressInput,
+        secAdressInput,
+        savedFirstAdress,
+        savedSecAdress,
+        savedName,
+        editButton,
+        contentSection,
+        buttonsBar,
+        stepContainer,
+        radioButton,
+        dialogContainer,
+        continueButton,
+        cancelButton
+    } = wizardPage;
+
+    beforeAll(() => {
+        wizardPage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForElDisplayed(wizardPage.title);
+    }, 2);
+
+    it('should check open-closing wizard', () => {
+        checkOpenClose(defaultExample);
+        checkOpenClose(customizableExample);
+        checkOpenClose(branchingExample);
+        checkOpenClose(ngforExample);
+        checkOpenClose(dialogExample)
+    });
+
+    it('should check basic way through default example', () => {
+        click(defaultExample + button);
+        const stepsLength = getElementArrayLength(wizard + step);
+
+        for (let i = 0; i < stepsLength; i++) {
+            i !== stepsLength - 1 ? expect(getElementClass(wizard + step, i + 1)).toContain('step--upcoming') : '';
```

Comment:
same; better to not use ternary if you don't have actions for both cases. 
if (i !== stepsLength - 1) {
expect...
}